examples/quickstart/jupyter-notebooks/docker-jupyter/docker-compose-local.yaml

```diff
@@ -69,7 +69,7 @@ services:
       - KAFKA_ENABLE_KRAFT=false
 
   coordinator:
-    image: apache/druid:${DRUID_VERSION}
+    image: apache/druid:${DRUID_VERSION:-26.0.0}
```

Comment:
hmm, what's the benefit of having a static default version here besides having to specify fewer command line args with docker-compose? Do we expect a developer to bump up this default version with every release as documented [here](https://github.com/apache/druid/blob/master/distribution/asf-release-process-guide.md#preparing-the-master-branch-for-the-next-version-after-branching)? For example, currently, it should be `28.0.0` to get the latest and greatest features

Reply:
@abhishekrb19 , The goal is fewer command line arguments, yes. This is only used by the tutorials, and we need to reduce friction as much as possible.

The tutorials do not run against unreleased versions like 28.0.0. Master should change to 27.0.0 at release time -- this can be a docs task.

These artifacts don't go into the distribution, so if someone downloads 27.0.0, won't include files that refer to 26.0.0. 

We'll update the [docs](https://druid.apache.org/docs/latest/tutorials/tutorial-jupyter-docker.html) to clarify in another pr.

Reply:
Thanks for the clarification, @techdocsmith. That sounds reasonable to me, especially if we have an automated way/process to bump up these versions _after_ a release, so folks playing around with notebooks always get the latest Druid version by default.